include/mxnet/c_api.h

```diff
@@ -1993,45 +1922,44 @@ MXNET_DLL int MXSymbolInferTypePartial(SymbolHandle sym,
  * \param dev_type device type
```

Comment:
This fragment and many other should not be formatted in this way

Reply:
This is caused probably because line is to long and when formatter breaks single /param  or /brief line into two or more lines than this issue happens

Reply:
Good catch! Just let me look at the problem closer!